### ISSUE TITLE
Woobly windows for floating layout?

### ISSUE THREAD

**Issue by System-x64** [hyprwm/Hyprland#906]
I was wondering, since hyprland is praised for its animations, won't adding woobly windows (like in KDE or wayfire) to floating layout be cool? I guess it's possible since wayfire is wlroots based aswell?

**Comment by vaxerski** [comment]
not planned for now

**Comment by System-x64** [comment]
Yeah actually, first let's just fix all the high priority stuff..maybe then we should work on appearance...
